test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
There are still spurious and unrelated changes in this file besides turning the dictionary into a list of tuples and removing calls to `dict.items()`